qiskit/primitives/base_estimator.py

```diff
@@ -147,7 +147,7 @@ def __init__(
                 will be bound. Defaults to ``[circ.parameters for circ in circuits]``
                 The indexing is such that ``parameters[i, j]`` is the j-th formal parameter of
                 ``circuits[i]``.
-            run_options: runtime options.
+            options: runtime options.
```

Comment:
Do we want to update this comment? For me these were options for the run() method, yes runtime is main aspect for primtives, but they can be elsewhere such as the ref ones here, in Aer, 3rd party...

Reply:
That's right. As you said, I think the use of options depends on subclasses. As this option of the base class is in the implementation of the run method, there is a default role for the run option, but there are no other restrictions. I'll simply write this as an option and leave it to the subclass how it will be used.